form/form.go

```diff
@@ -3,6 +3,9 @@
 
 // Package form provides ways to create and process forms based on
 // environschema schemas.
+//
+// The API exposed by this package is not currently subject
+// to the environschema.v1 API compatibility guarantees.
```

Comment:
because it's still in active development and we're breaking everything all over the place in this PR and probably will in near-future PRs too.